src/components/NavBar.module.css

```diff
@@ -0,0 +1,55 @@
+.root {
+  --icon-font: montserrat;
+  --font: robotoslab;
+}
+
+.NavBar {
+  display: flex;
+  justify-content: flex-start;
+  padding: 2px 5px;
+  align-items: center;
+  height: 3rem;
+  margin-bottom: 1rem;
+  background-color: #fff;
+  margin-top: 1rem;
+}
```

Comment:
- [ ] Please adjust the styling of the Navbar to be separated from the main section in order to be exactly as the [original design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). 😄 🙏 

![Screenshot (1)](https://user-images.githubusercontent.com/47625407/132089174-0b965112-0889-4390-b68d-9e4f888fb649.png)